src/shared/install.c

```diff
@@ -3771,7 +3800,9 @@ int unit_file_get_list(
                             !strv_contains(states, unit_file_state_to_string(f->state)))
                                 continue;
 
-                        r = hashmap_put(h, basename(f->path), f);
+                        /* It was recommended to not interact with */
+
+                        r = hashmap_put(h, f->path, f);
```

Comment:
The comment is truncated, and this change doesn't look right. Just leave this basename() in place for now and avoid changing this part.

Reply:
there's been a very drastic change in my code after trying to rebase and squash.  It ruined the commits I previously did and i don't even know why. but it's a mess now, so now i have to go back and change everything

Reply:
https://www.git-tower.com/learn/git/faq/what-is-git-reflog